dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -1734,3 +1738,28 @@ def test_diff(self):
             str(exc.exception),
             "Unsupported operand type(s) for diff: 'FloatColumn' and" " 'str'",
         )
+
+    def test_json_decode(self):
+        fake_profile_name = "Fake profile name"
+        # Actual deserialization
+
+        # Build expected CategoricalColumn
+        df_categorical = pd.Series(
+            [
+                1.2,
+                1.4,
+                5.232,
+                8.4234,
+                2.32,
+                5.1,
+            ]
+        ).apply(str)
+        expected_profile = FloatColumn(fake_profile_name)
+
+        with mock.patch("time.time", side_effect=lambda: 0.0):
+            expected_profile.update(df_categorical)
+
+        serialized = json.dumps(expected_profile, cls=ProfileEncoder)
+        deserialized = load_column_profile(serialized)
+
+        test_utils.assert_profiles_equal(deserialized, expected_profile)
```

Comment:
We might want to update the exceptions in this func to be asserts / be more descriptive such that we can get good insights into errors.